migration/sql-files/029-add-space-id-wilt.sql

```diff
@@ -0,0 +1,6 @@
+-- Alter the tables
+ALTER TABLE work_item_link_types ADD space_id uuid;
+ALTER TABLE work_item_link_types ADD FOREIGN KEY (space_id) REFERENCES spaces(id);
```

Comment:
Now I know we don't do hard deletes but you should add an `ON DELETE CASCADE` anyways.